Header header
# sensor_msgs/Image image
uint32 image_w
uint32 image_h
Person[] persons